addon/components/polaris-action-list.js

```diff
@@ -1,74 +1,50 @@
-import Component from '@ember/component';
-import { computed } from '@ember/object';
+import Component from '@glimmer/component';
 import { gt } from '@ember/object/computed';
-import { isPresent } from '@ember/utils';
-import { isArray } from '@ember/array';
-import { assert } from '@ember/debug';
-import { tagName, layout } from '@ember-decorators/component';
-import template from '../templates/components/polaris-action-list';
 
 /**
  * Polaris action list component.
  * See https://polaris.shopify.com/components/actions/action-list
+ *
+ * @version 5.6.0
  */
-@tagName('')
-@layout(template)
-export default class PolarisActionList extends Component {
+export default class PolarisActionListComponent extends Component {
   /**
    * Collection of actions for list
-   *
    * @type {Array}
-   * @default []
-   * @public
    */
-  items = [];
+  items;
```

Comment:
Same comment for these as [here](https://github.com/smile-io/ember-polaris/pull/664/files#r503747187) re. the value of declaring these if we're not assigning default values 🤔